src/mat.rs

```diff
@@ -1267,10 +1267,10 @@ macro_rules! mat4s {
                 let vec2 = $vt::new(m12, m02, m02, m02);
                 let vec3 = $vt::new(m13, m03, m03, m03);
 
-                let inv0 = vec1.mul_add(fac0, -(vec2.mul_add(fac1, -(vec3 * fac2))));
-                let inv1 = vec0.mul_add(fac0, -(vec2.mul_add(fac3, -(vec3 * fac4))));
-                let inv2 = vec0.mul_add(fac1, -(vec1.mul_add(fac3, -(vec3 * fac5))));
-                let inv3 = vec0.mul_add(fac2, -(vec1.mul_add(fac4, -(vec2 * fac5))));
+                let inv0 = (vec1 * fac0) - ((vec2 * fac1) - (vec3 * fac2));
```

Comment:
I can go through and fix these after merging if you like, thought I'd make a note in either case -- this can be expanded to remove the double negation

```
let inv0 = (vec1 * fac0) - (vec2 * fac1) + (vec3 * fac2);
```